pxtsim/simdriver.ts

```diff
@@ -529,7 +533,7 @@ namespace pxsim {
             if (!this.listener) {
                 this.listener = (ev: MessageEvent) => {
                     if (this.hwdbg) return
-                    this.handleMessage(ev.data, ev.source)
+                    this.handleMessage(ev.data, ev.source as Window)
```

Comment:
This change is unrelated, just casting explicitly as it's missing unnecessary properties of window and showing up as an error in the sidebar without the cast